pkg/buildkitutil/buildkitutil.go

```diff
@@ -181,3 +185,42 @@ func WriteTempDockerfile(rc io.Reader) (dockerfileDir string, err error) {
 	}
 	return dockerfileDir, nil
 }
+
+// Buildkit file returns the values for the following buildctl args
+// --localfilename=dockerfile={absDir}
+// --opt=filename={file}
+func BuildKitFile(dir, inputfile string) (absDir string, file string, err error) {
+	file = inputfile
+	if file == "" || file == "." {
+		file = DefaultDockerfileName
+	}
+	absDir, err = filepath.Abs(dir)
+	if err != nil {
+		return "", "", err
+	}
+	if file != DefaultDockerfileName {
+		if _, err := os.Lstat(filepath.Join(absDir, file)); err != nil {
+			return "", "", err
+		}
+	} else {
+		dockerfile, dockerfileerr := ioutil.ReadFile(filepath.Join(absDir, file))
+		containerfile, containerfileerr := ioutil.ReadFile(filepath.Join(absDir, ContainerfileName))
```

Comment:
Only need to read the file content if both files exist. Using `os.Lstat` replace it.